maquetado/index.html

```diff
@@ -8,6 +8,21 @@
  <link rel="stylesheet" type="text/css" href="scss/out/application.css">
 </head>
 <body>
-  <h1 class="title">Hello world</h1>      
+  <div>
+    <p class="text-big text-bold book-element">Título del libro</p>
```

Comment:
This should be an `h2` tag since we are talking about a title. We can't or shouldn't set `h1` though 'cause it's not the page's main title.